docs/src/main/asciidoc/security-proactive-authentication.adoc

```diff
@@ -110,6 +110,51 @@ However, some CDI beans may not be ready for use.
 One exception to this solution is a situation when application endpoints are secured with the xref:security-authorize-web-endpoints-reference.adoc#authorization-using-configuration[Authorization using configuration].
 For more information, see the xref:security-authorize-web-endpoints-reference.adoc#policy-active-cdi-request-context[Inject RequestScoped beans into HttpSecurityPolicy] section of the "Authorization of Web endpoints" guide for more information.
 
+=== Make `SecurityIdentity` available in newly activated CDI request context
+
+There are infrequent situations where Quarkus cannot populate CDI request context with current `SecurityIdentity` because you activated CDI request context before Quarkus could prepare it.
```

Comment:
I'm not sure it will not be confusing, as  a user I just register a Vertx handler directly in the router and it won't work but it does work if register the route with `@Route`.

IMHO it is worth replacing both lines 115 and 116 with something like 
```
Currently, if you need to inject `SecurityIdentity` in a Vert.x route handler which is registered directly on the xref:reactive-routes.adoc#using-vertx-web-router[Vert.x Web Router], then requesting the `SecurityIdentity` propagation is required. 
For example:
```

and then give this example